libs/core/src/components.d.ts

```diff
@@ -47,15 +74,37 @@ declare namespace LocalJSX {
          */
         "middle"?: string;
     }
+    interface SageLink {
```

Comment:
I understand this is in a different block, but why do we _need_ to define this information twice?

Reply:
This file is autogenerated

Reply:
Also, an `interface` is different than a `class`. An `interface` is a contract stating what should be in a class that inherits the class.